### DISCUSSION TITLE
Consequences : Resources, Tools, Expansions & Stock Proposal

### DISCUSSION THREAD

**Discussion by alivitedg** [sunflower-land/sunflower-land#1962] [General]
there is a lot of consequences if you implement this proposal.

the problem is about new player.

the big problem to new player is the big number of sync you need to augment number of axe can player buy it from wrokbunch to 200 or 300.

then you need to reduce recource needed to expand first's land to 50%.
not all expand's just some first expand's.

don't change economic system and augment base gathering to 1.25 and reduce pickaxe wood from 5 to 2 because if you do this all gathering nft price down and any new player not need any gathering nft to feel the difference. because wihout nft he can do anything easy and in a very short period of time.
also all resource lose values like gold hi's value go to 1 to 3 sfl very low, and inflation will return.
there is a number of players read this decision sell his nft and quit game also the big number of sync to buy axe make some new players to quit game.

**Comment by adamhannigan** [comment]
Thanks for your feedback.

We recently opened a discussion around this proposal - https://github.com/sunflower-land/sunflower-land/discussions/1952

Can you please add any points there so others can see them
